python/social_bot/envs/grocery_ground.py

```diff
@@ -141,10 +170,20 @@ def __init__(self,
         super(GroceryGround, self).__init__(port=port)
         self._world = gazebo.new_world_from_file(
             os.path.join(social_bot.get_world_dir(), "grocery_ground.world"))
-        self._object_types = [
-            'coke_can', 'table', 'bookshelf', 'cube_20k', 'car_wheel',
-            'plastic_cup', 'beer', 'hammer'
-        ]
+
+        self._teacher = teacher.Teacher(False)
```

Comment:
Use a keyword for "False" to make it easier to read the code.

Reply:
Done. It's task_groups_exclusive=False .